website/docs/r/scheduled_view.html.markdown

```diff
@@ -21,18 +21,25 @@ _view=connections connectionStats
 QUERY
   start_time = "2019-09-01T00:00:00Z"
   retention_period = 365
+  lifecycle {
+    prevent_destroy = true
+  }
 }
 ```
 
 ## Argument reference
 
 The following arguments are supported:
 
-- `index_name` - (Required) Name of the index (scheduled view).
-- `query` - (Required) Log query defining the scheduled view. This value cannot be updated.
-- `start_time` - (Required) Starting date/time for log indexing.
-- `retention_period` - (Optional) Number of days to keep the scheduled view data for.
-- `data_forwarding_id` - (Optional) ID of a data forwarding configuration to be used by the scheduled view.
+Note: For attributes that force a new resource, if the value is updated, it will destroy the resource and recreate it which may incur significant costs. We advise customers to set the `lifecycle` attribute `prevent_destroy` to `true` to avoid accidentally destroying and recreating expensive resources.
```

Comment:
Use `~> ` instead of Note, like  `~> For attributes...`. It will show up nicely in the documentation. You can check out the preview here https://registry.terraform.io/tools/doc-preview

Reply:
Oooh that is cool. Thanks!